pom.xml

```diff
@@ -279,7 +276,7 @@
 
     <profiles>
         <profile>
-            <id>release</id>
+            <id>apache-release</id>
```

Comment:
@coderzc @zyxxoo @z7658329 
seems we should use `apache-release` id here to override the settings when execute `mvn xx -Papache-release`